### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by joindevopscloud**

### CODE REVIEW

**Comment by sivakumarreddym** [comment]
why you added oil to the karam dosa? already oil is there for plain dosa? why you added specifically

**Comment by joindevopscloud** [comment]
> why you added oil to the karam dosa? already oil is there for plain dosa? why you added specifically
extra oil is good for karam dosa

File: menu.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-dosa
\ No newline at end of file
+dosa
+karam
+oil
```
Comment (sivakumarreddym) [inline menu.txt:3]: why oil is added at the end?

**Review by sivakumarreddym — APPROVED** [review]
everything is good, we can go ahead

**Pull request merged**